reconcile/utils/oc.py

```diff
@@ -514,6 +522,32 @@ def get_owned_pods(self, namespace, resource):
 
         return owned_pods
 
+    @staticmethod
+    def get_pod_owned_pvc_names(pods):
+        owned_pvc_names = set()
+        for p in pods:
+            vols = p.get('volumes')
+            if not vols:
+                continue
+            for v in vols:
+                with suppress(KeyError):
+                    cn = v['persistentVolumeClaim']['claimName']
+                    owned_pvc_names.add(cn)
+
+        return owned_pvc_names
+
+    @staticmethod
+    def get_storage(resource):
```

Comment:
In this case, it looks like the type hint is
```suggestion
    def get_storage(resource: dict) -> Optional[str]:
```

Reply:
thanks!

Reply:
this caused a failure due to a missing return statement. reverted it.